packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -409,6 +409,7 @@ export type BookingHandlerInput = {
   // These used to come from headers but now we're passing them as params
   hostname?: string;
   forcedSlug?: string;
+  traceContext?: TraceContext;
```

Comment:
This is called by API v1, web app etc. so making it optional for now. In a follow up PR this would also become required after some testing